presto-product-tests/src/main/java/com/facebook/presto/tests/JmxConnectorTests.java

```diff
@@ -31,7 +32,7 @@
 public class JmxConnectorTests
         extends ProductTest
 {
-    @Test(groups = JMX_CONNECTOR)
+    @Test(groups = {JMX_CONNECTOR, SIMBA_JDBC})
```

Comment:
maybe put it into just `jdbc`, it tests also public jdbc. Please change it for other tests as well.